package.json

```diff
@@ -11,41 +11,37 @@
     "clean": "run-p --print-label clean:**",
     "build": "run-s --print-label build:api build:api-db-mongodb build:api-media-karma build:oauth2 build:editor build:oauth2 build:api-example",
     "build:demo": "run-s --print-label build:api build:api-db-mongodb build:api-media-karma build:oauth2 build:editor build:api-example build:media-example build:website-example",
+    "preversion": "run-s clean build",
+    "verison": "lerna version --conventional-commits --create-release github --force-git-tag",
+    "publish": "lerna publish from-package --ignore-scripts",
     "watch": "run-p --print-label watch:*",
     "start:docker": "docker-compose up mongo mongo-express",
     "clean:api": "yarn workspace @wepublish/api clean",
     "build:api": "yarn workspace @wepublish/api build",
     "watch:api": "yarn workspace @wepublish/api watch",
-    "publish:api": "yarn workspace @wepublish/api publish --access public --no-git-tag-version",
```

Comment:
What did you do in this file in general? Removing no longer needed scripts?

Reply:
yes. With lerna we don't need the publish script anymore.